admin/pages/index.tsx

```diff
@@ -62,7 +62,10 @@ export default function Home() {
                 "m-auto": !configured,
               })}
             >
-              <ModelConfig setConfigured={setConfigured} />
+              <ModelConfig
+                configured={configured ?? false}
+                setConfigured={setConfigured}
+              />
```

Comment:
```suggestion
              <ModelConfig
                 title={configured ? "Update model" : "Start"}
      description={
        configured
          ? "Change to a different model or use another provider"
          : "Set up an AI model to start the app."
      }
                onConfigChange={handleModelConfigChange}
              />
```

and in `handleModelConfigChange` you retrigger `fetchIsAppConfigured()`